provider/implementations/py/polywrap_ethereum_provider/networks.py

```diff
@@ -0,0 +1,39 @@
+"""This module contains a list of known networks."""
+from enum import IntEnum, unique
+from typing import List
+
+
+@unique
+class KnownNetwork(IntEnum):
+    """Defines a list of known networks."""
+    mainnet = 1, "1", "mainnet"
+    goerli = 5, "5", "goerli"
+    sepolia = 11155111, "11155111", "sepolia"
+    celo_mainnet = 42220, "42220", "celo_mainnet"
+    celo_alfajores = 44787, "44787", "celo_alfajores"
+    avalanche_mainnet = 43114, "43114", "avalanche_mainnet"
+    avalanche_fuji = 43113, "43113", "avalanche_fuji"
+    palm_mainnet = 11297108109, "11297108109", "palm_mainnet"
+    palm_testnet = 11297108099, "11297108099", "palm_testnet"
+    aurora_mainnet = 1313161554, "1313161554", "aurora_mainnet"
+    aurora_testnet = 1313161555, "1313161555", "aurora_testnet"
```

Comment:
seeing this makes me wonder, should we create a JSON that is shared between different provider implementations :thinking: ? just for the sake of maintainability and consistency between different implementations

Reply:
or maybe we can add a function of getting the network to the wrapper and use this from ethers rs https://github.com/Oighty/ethers-rs/blob/e1854ad0d13ef68a695ced81b4d11a084aee4e5d/ethers-core/src/types/chain.rs#L55

Reply:
No packaging json may not be as easy as js for other languages. If we really want to have a common list then we should declare this in the schema.